src/gpu/ganesh/mtl/GrMtlCaps.mm

```diff
@@ -96,7 +96,7 @@
             return true;
         }
     }
-#elif defined(SK_BUILD_FOR_IOS)
+#elif defined(SK_BUILD_FOR_IOS) && !defined(SK_BUILD_FOR_TVOS)
```

Comment:
Does it mean that we have both variables at the same time?

Reply:
Yes, Skia build scripts defines `SK_BUILD_FOR_IOS` when targeting tvOS, and we will define `SK_BUILD_FOR_TVOS` externally to just skip this part. Later we could do a better job if we want to contribute to the Skia itself, maybe. All over the Skia code it uses `SK_BUILD_FOR_IOS` for Metal and stuff.